lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -369,6 +369,7 @@ def add_transaction_data(post, options = {})
       end
 
       def merchant_initiated_override(post, options)
+        post[:payment_type] = 'Regular' unless post[:payment_type]
```

Comment:
I suggest you change the unless condition for the double pipe:

```suggestion
        post[:payment_type] ||= 'Regular'
```